spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -251,6 +258,74 @@ def compute_isi_violations(waveform_extractor, isi_threshold_ms=1.5, min_isi_ms=
     return res(isi_violations_ratio, isi_violations_rate, isi_violations_count)
 
 
+def compute_refrac_period_violations(waveform_extractor, refractory_period: tuple = (0.0, 1.0)):
+    """Calculates the number of refractory period violations.
+
+    This is similar (but slightly different) to the ISI violations.
+    The key difference being that the violations are not only computed on consecutive spikes.
+
+    This is required for some formulas (e.g. the ones from Llobet & Wyngaard 2022).
+
+    Parameters
+    ----------
+    waveform_extractor : WaveformExtractor
+        The waveform extractor object
+    refractory_period : tuple of 2 floats, optional, default: (0.0, 1.0)
+        This tuple contains the censored period as well as the refractory period (in ms).
+        If there is no censored period, you can leave it at 0.0.
```

Comment:
Theses parameters are not intuitive.
Maybe you make 2 varaible name with more explanation.
Espatially for the first one.